specs/payment_version.md

```diff
@@ -0,0 +1,17 @@
+### Added possibility of setting version of payment protocol

```

Comment:
New entry in `Golem Capability List` should be added mentioning ability to validate multi-payments